tests/integration_test/algorithm_options.py

```diff
@@ -36,14 +36,18 @@ def apply_remove_twins(G: "Graph"):
     return factor
 
 
-def apply_tree_algorithm(G: "Graph", H: "Graph", count_isomorphisms: "Bool"):
+def apply_tree_algorithm(G: "Graph", H: "Graph" = None):
     """
-    If the tree algorithm must be used, this method checks if G and H are trees.
+    If the tree algorithm must be used, this method checks if G (and H) are trees.
     If they are the isomorphism problem is resolved using the tree isomorphism method.
     """
     if tree_algorithm:
-        if is_tree(G) and is_tree(H):
-            return True, trees_count_isomorphisms(G, H, count_isomorphisms)
+        if H is None and is_tree(G):
+            # Count automorphism problem
+            return True, trees_automorphisms(G)
+        elif is_tree(G) and is_tree(H):
+            # Check if G and H are isomorph
+            return True, trees_are_isomorph(G, H)
```

Comment:
Er staat niet in de documentatie wat de return waarden zijn. Ben er wel uitgekomen, maar denk wel goed om te laten zien.

Reply:
Heb documentatie toegevoegd